### PR TITLE
Using SweetAlert over AngularJS factory

### PR DESCRIPTION
**Pull request by fiftin**

### CODE REVIEW

File: web/public/js/factories/sweetalert.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+app.factory('SweetAlert', [ '$rootScope', function ( $rootScope ) {
+	return {
+		swal: function () {
```
[the remark this answers is not in the document]
Comment (fiftin) [reply web/public/js/factories/sweetalert.js]: Web should use ES6?

[the remark this answers is not in the document]
Comment (twhiston) [reply web/public/js/factories/sweetalert.js]: Honestly I'm not sure. I still think we need to pick an FE styleguide and do the work to make everything conform to it as at the moment there are no real 'rules' about the FE code.

**Comment by twhiston** [comment]
can you give me some context about this change please? Looks good though and thanks for your work on making things windows compatible!

**Comment by twhiston** [comment]
Tests fail due to mysql issues, see #536

**Comment by twhiston** [comment]
@fiftin - please see my comment above, looks good but why do this? can you give a quick update on what the reasoning is.

**Comment by fiftin** [comment]
To avoid "'swal' is not defined" warning in Codacy

**Comment by twhiston** [comment]
:+1: ok cool, thank you! If you can fix that merge conflict i'll get it straight in

**Comment by twhiston** [comment]
ignoring build fail, due to ci gpg key usage issue, see #541.
Approved

**Pull request merged**